metalearn/metafeatures/constants.py

```diff
@@ -0,0 +1,27 @@
+from enum import Enum
+
+# Constant Enums
+class ProblemType(Enum):
+    CLASSIFICATION = "classification"
+    REGRESSION = "regression"
+    ANY = "any"
+
+class MetafeatureGroup(Enum):
+    ALL = "all"
+    SIMPLE = "simple"
+    TEXT = "text"
+    STATISTICAL = "statistical"
+    INFO_THEORETIC = "info_theoretic"
+    LANDMARKING = "landmarking"
+    MODEL_BASED = "model_based"
+    TARGET_DEPENDENT = "target_dependent"
+
+# Constant strings
+VALUE_KEY = 'value'
+COMPUTE_TIME_KEY = 'compute_time'
+NUMERIC = "NUMERIC"
```

Comment:
I was torn on this idea. Enums are clean and more strict, but when you want to use the actual string of an Enum, i.e. if you want to use the string value `"NUMERIC"`, you have to say `ColumnType.NUMERIC.value`, rather than `consts.NUMERIC`. We could restructure all uses of `column_types` to reference `ColumnType` enum members instead of strings, and just say `ComputeErrors.NO_TARGETS.value` in the case of compute errors, but I think that would be best as a separate PR and issue. I'd love to have your thoughts on this though.

Reply:
This discussion is outside the scope of the PR and has been moved to a new issue: #198.